DeepSkyStacker/DeepStack.cpp

```diff
@@ -56,7 +56,7 @@ void CDeepStack::ComputeOriginalHistogram(CRGBHistogram & Histo)
 
 	if (!m_StackedBitmap.IsMonochrome())
 	{
-#pragma omp parallel sections default(none) shared(redPixels, greenPixels, bluePixels) if(nrEnabledThreads - 1)
+#pragma omp parallel sections default(none) shared(Histo, redPixels, greenPixels, bluePixels) if(nrEnabledThreads - 1)
```

Comment:
The compiler now returns an error if we don't specify the data-sharing attribute of `Histo`. I set it as `shared` as I believe it is what it defaulted to before, and the parallel sections below _seem_ to me like they are thread-safe, but I'm not 100% sure.